include/grpc/support/time.h

```diff
@@ -47,7 +47,11 @@ extern "C" {
 
 typedef struct gpr_timespec {
     time_t tv_sec;
+#if defined(GPR_WIN32) && defined(GPR_ARCH_64)
```

Comment:
But while the typecastings should raise a warning, they shouldn't be erroneous, because mathematically, we know that the data fits.